test/functionalTest/cases/0970_create_entity/errors_at_creation.test

```diff
@@ -0,0 +1,351 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Entities V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 00. POST /v2/entities with Entity::id AND Entity::idPattern
+# 01. POST /v2/entities without Entity::id
+# 02. POST /v2/entities with empty Entity::id
+# 03. POST /v2/entities with empty Entity::idPattern
+# 04. POST /v2/entities with boolean Entity::id
+# 05. POST /v2/entities with integer Entity::id
+# 06. POST /v2/entities with boolean Entity::type
+# 07. POST /v2/entities with integer Entity::type
+# 08. POST /v2/entities without Attribute::name
+# 09. POST /v2/entities with integer Attribute::name
+# 10. POST /v2/entities with boolean Attribute::name
+# 11. POST /v2/entities with empty Attribute::value
+#
+
+echo "00. POST /v2/entities with Entity::id AND Entity::idPattern"
+echo "==========================================================="
+payload='{
+  "id": "id",
+  "idPattern": "pattern",
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "01. POST /v2/entities without Entity::id"
+echo "========================================"
+payload='{
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "02. POST /v2/entities with empty Entity::id"
+echo "==========================================="
+payload='{
+  "id": "",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "03. POST /v2/entities with empty Entity::idPattern"
+echo "=================================================="
+payload='{
+  "idPattern": "",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "04. POST /v2/entities with boolean Entity::id"
+echo "============================================="
+payload='{
+  "id": true,
+  "type": "T1",
+  "attr01": "1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "05. POST /v2/entities with integer Entity::id"
```

Comment:
Yeah, but in this case it is simply an integer.
It doesn't have anything to do really with whether we use Numbers of int/float in our implementation.
"with number Entity::id" would be very incorrect English.
I will leave this as is.

NTC